### PR TITLE
chore: Update MomentoSDK to the latest

### PR DESCRIPTION
**Pull request by poppoerika**

### CODE REVIEW

File: dotnet/MomentoExamples/MomentoApplication/MomentoApplication.sln [hunk 1]
```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.810.16
```
Comment (tylerburdsall) [inline dotnet/MomentoExamples/MomentoApplication/MomentoApplication.sln:4]: Do solution files usually get committed?

**Review by tylerburdsall — COMMENTED** [review]
Example looks good! I think the solution file accidentally got committed. It's been a while since I've done C# development so I can't remember if we actually commit those

File: dotnet/MomentoExamples/MomentoApplication/MomentoApplication.sln (hunk 1 above)
Comment (poppoerika) [reply dotnet/MomentoExamples/MomentoApplication/MomentoApplication.sln:4]: I'm not sure either... but I see that `.sln` file was committed [here](https://github.com/momentohq/client-sdk-examples/commit/dc12441710e21808d1e5cb2e861e3a276a92098c#diff-b0e83dadbdf9295c80998742d16774abff392128eabb324043413ca85eaf8306) 🤔

Comment (tylerburdsall) [reply dotnet/MomentoExamples/MomentoApplication/MomentoApplication.sln:4]: It looks like [this answers the question](https://stackoverflow.com/questions/1033809/should-a-sln-be-committed-to-source-control#:~:text=Yes%20you%20should%20do%20this,contain%20any%20user%20specific%20settings.), so it should be okay 👍

**Review by tylerburdsall — APPROVED** [review]

**Pull request merged**